MobileWallet/UIElements/TransactionDetailNavigationBar.swift

```diff
@@ -0,0 +1,147 @@
+//  TransactionDetailNavigationBar.swift
+
+/*
+    Package MobileWallet
+    Created by S.Shovkoplyas on 16.04.2020
+    Using Swift 5.0
+    Running on macOS 10.15
+
+    Copyright 2019 The Tari Project
+
+    Redistribution and use in source and binary forms, with or
+    without modification, are permitted provided that the
+    following conditions are met:
+
+    1. Redistributions of source code must retain the above copyright notice,
+    this list of conditions and the following disclaimer.
+
+    2. Redistributions in binary form must reproduce the above
+    copyright notice, this list of conditions and the following disclaimer in the
+    documentation and/or other materials provided with the distribution.
+
+    3. Neither the name of the copyright holder nor the names of
+    its contributors may be used to endorse or promote products
+    derived from this software without specific prior written permission.
+
+    THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND
+    CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES,
+    INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES
+    OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+    DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR
+    CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+    SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT
+    NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+    LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+    HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+    CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE
+    OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+    SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+*/
+
+import UIKit
+
+protocol TransactionDetailNavigationBarProtocol: class {
+    var title: String? { get set }
+    var subtitle: String? { get set }
+}
+
+class TransactionDetailNavigationBar: UIView, TransactionDetailNavigationBarProtocol {
```

Comment:
Is the reason for implementing this custom navbar just for the subtitle label? I think maybe if we can rename it to something more generic as we'll need to use this in a lot of places. Would rather not have to do nav bar updates in multiple components

Reply:
Not only for subtitle. Also for correct place for back button. On mockups we can see specific location for it. And the most important reason for custom nav bar it is possibility to animate it with our rules. But i can agree with you about naming this class and reusable possibility it.

Reply:
Ok great, yes I think just rename so we can start using it in other places as well